app/Services/LanguagePackageServiceProvider.js

```diff
@@ -8,16 +9,24 @@ async function createLanguagePackage(
   { name, foreignWordLanguage, translatedWordLanguage, vocabsPerDay, rightWords },
   userId
 ) {
-  const languagePackage = await LanguagePackage.create({
-    userId,
-    name,
-    foreignWordLanguage,
-    translatedWordLanguage,
-    vocabsPerDay,
-    rightWords,
-  });
+  try {
+    const languagePackage = await LanguagePackage.create({
+      userId,
+      name,
+      foreignWordLanguage,
+      translatedWordLanguage,
+      vocabsPerDay,
+      rightWords,
+    });
 
-  return deleteKeysFromObject(['userId', 'createdAt', 'updatedAt'], languagePackage.toJSON());
+    return deleteKeysFromObject(['userId', 'createdAt', 'updatedAt'], languagePackage.toJSON());
+  } catch (error) {
+    if (error instanceof ForeignKeyConstraintError) {
```

Comment:
I've already catched this here, to send a specific message so the user knows what it's about. And if it isn't a `ForeignKeyConstraintError` I throw the error, so it will be handeld in the Middleware.

Reply:
Ok